src/Scope.js

```diff
@@ -0,0 +1,152 @@
+import { blank, keys } from './utils/object';
+
+// A minimal `Identifier` implementation. Anything that has an `originalName`,
+// and a mutable `name` property can be used as an `Identifier`.
+function Identifier ( name ) {
```

Comment:
I'll adapt the code to whatever convention is preferred for Rollup's source code. I chose `function` since I thought it'd be clearest.